.gitmodules

```diff
@@ -52,3 +52,6 @@
 [submodule "system/tbb_sm/tbb"]
     path = system/tbb_sm/tbb
     url = https://github.com/hpcc-systems/tbb.git
+[submodule "esp/src/hpcc-js-comms"]
+	path = esp/src/hpcc-js-comms
+	url = https://github.com/GordonSmith/hpcc-js-comms.git
```

Comment:
Change to use https://github.com/hpcc-systemshpcc-js-comms.git once https://github.com/hpcc-systems/hpcc-js-comms/pull/2 is pulled.